conformance/fixtures/CV-CHAINSTATE.yml

```
@@ -0,0 +1,60 @@
+          covenant_data: "000000000000000000"
+          creation_height: 0
+          created_by_coinbase: false
+      blocks_hex:
+        - "0100000057b75170b9a0fafba6bbb5d9f462b2ddd66bef6154a991a733e128e3b1362def81d4be75eb679e13e15a42b2f7ce16ab5ec363f525fa3eea14d368da6cad9e9201f1536500000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff000000000000000002010000000000000000000000010000000000000000000000000000000000000000000000000000000000000000ffffffff00ffffffff01010000000000000000002101000000000000000000000000000000000000000000000000000000000000000001000000000100000007000000000000000111111111111111111111111111111111111111111111111111111111111111110000000000010000000101000000000000000000210100000000000000000000000000000000000000000000000000000000000000000000000001000000"
+        - "010000001777aae4c594a650f78e5563d7c6eda5e08acbd0432ad4787ccbbff9eeb7f3f18b7bd3f0e9594e14f108a47b9b6a73f44bd8635278f9963088f13f31e512444f02f1536500000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff000000000000000001010000000000000000000000010000000000000000000000000000000000000000000000000000000000000000ffffffff00ffffffff0101000000000000000000210100000000000000000000000000000000000000000000000000000000000000000200000000"
+    expected_code: PASS
+    expected_tip_height: 2
+    expected_tip_hash_hex: "d72a68b0e749ed2ba884a14791c91d042efe4fd0be9a75e56caa77ca78ecc800"
+    expected_utxo_set_hash_hex: "a20cf249c6651935a078322b4e25ce98b126f3b89b56da8f247443dcf5ad8add"
+    consensus: true
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)